src/app/page/Signup/Signup.js

```diff
@@ -7,31 +7,70 @@ import Button from "../../component/Button";
 import { FormWrapper, HeaderText, DescriptionText, PaddedDiv } from "./Signup.style";
 
 const SignupPage = () => {
+  const [form, setForm] = useState({
+    fullName: "",
+    email: "",
+    phoneNumber: "",
+    zipCode: "",
+    errors: "",
+  });
+
   const handleLoginCTAClick = (e) => {
     e.preventDefault();
 
     window.location = "/login";
   };
 
-  const handleFormSubmit = () => {
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
     console.log("Submit form and redirect user");
+    let newState = {};
+
+    for (let field in form) {
+      if (!form.field) newState[field] = "REQUIRED";
+    }
+
+    setForm(newState);
+  };
+
+  const handleChange = (e) => {
+    setForm({
+      ...form,
+      [e.target.name]: e.target.value,
+    });
   };
 
   return (
     <Page>
       <FormWrapper>
         <HeaderText>Create account</HeaderText>
         <DescriptionText>Create an account to post your request</DescriptionText>
-        <form onSubmit={() => handleFormSubmit()}>
+        <form onSubmit={(e) => handleFormSubmit(e)}>
           <Row>
             <Col xs={12}>
               <PaddedDiv>
-                <Input inputType="text" dataId="fullName" inputName="fullName" label="FULL NAME" />
+                <Input
+                  inputType="text"
+                  dataId="fullName"
+                  inputName="fullName"
+                  label="FULL NAME"
+                  placeholder={form["fullName"]}
+                  value={form["fullName"]}
+                  onChange={(e) => handleChange(e)}
+                />
               </PaddedDiv>
             </Col>
             <Col xs={12}>
               <PaddedDiv>
-                <Input inputType="text" dataId="email" inputName="email" label="EMAIL" />
+                <Input
+                  inputType="text"
+                  dataId="email"
+                  inputName="email"
+                  label="EMAIL"
+                  placeholder={form["email"]}
+                  value={form["email"]}
+                  onChange={(e) => handleChange(e)}
```

Comment:
Since onChange function take in the same variable as the input function you can also do
                  onChange={(e) => handleChange(e)}
To 
                  onChange={handleChange}

Reply:
I didn't know that. I'll take care of it. Thanks